### ISSUE TITLE
TypeError: issubclass() arg 1 must be a class

### ISSUE THREAD

**Issue by ekiwi111** [aiwaves-cn/agents#95]
1. Cloned and installed with `pip install -e .`
2. Went to examples directory `cd examples`
3. Tried one of the examples with `EVENTLET_NO_GREENDNS="yes" python Single_Agent/run_gradio.py --agent Single_Agent/shopping_assistant/config.json` (Added `EVENTLET_NO_GREENDNS="yes"`  beacuse of https://github.com/aiwaves-cn/agents/issues/94#issuecomment-1734675019)

Got the following output:

```
server: executing `python Single_Agent/gradio_backend.py --agent Single_Agent/shopping_assistant/config.json` ...
[2023-09-26 13:58:06,608] [INFO] [real_accelerator.py:158:get_accelerator] Setting ds_accelerator to cuda (auto detect)
Traceback (most recent call last):
  File "/home/user/Developer/_experiments/agents/examples/Single_Agent/gradio_backend.py", line 7, in <module>
    from agents.SOP import SOP
  File "/home/user/Developer/_experiments/agents/src/agents/__init__.py", line 2, in <module>
    from .SOP import *
  File "/home/user/Developer/_experiments/agents/src/agents/SOP.py", line 18, in <module>
    from LLM.base_LLM import *
  File "/home/user/Developer/_experiments/agents/examples/../src/agents/LLM/base_LLM.py", line 6, in <module>
    from utils import save_logs
  File "/home/user/Developer/_experiments/agents/examples/../src/agents/utils.py", line 28, in <module>
    from langchain.document_loaders import UnstructuredFileLoader
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/__init__.py", line 6, in <module>
    from langchain.agents import MRKLChain, ReActChain, SelfAskWithSearchChain
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/agents/__init__.py", line 31, in <module>
    from langchain.agents.agent import (
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/agents/agent.py", line 23, in <module>
    from langchain.agents.agent_iterator import AgentExecutorIterator
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/agents/agent_iterator.py", line 30, in <module>
    from langchain.tools import BaseTool
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/tools/__init__.py", line 25, in <module>
    from langchain.tools.arxiv.tool import ArxivQueryRun
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/tools/arxiv/tool.py", line 8, in <module>
    from langchain.utilities.arxiv import ArxivAPIWrapper
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/utilities/__init__.py", line 7, in <module>
    from langchain.utilities.apify import ApifyWrapper
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/utilities/apify.py", line 3, in <module>
    from langchain.document_loaders import ApifyDatasetLoader
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/document_loaders/__init__.py", line 89, in <module>
    from langchain.document_loaders.github import GitHubIssuesLoader
  File "/home/user/anaconda3/lib/python3.9/site-packages/langchain/document_loaders/github.py", line 37, in <module>
    class GitHubIssuesLoader(BaseGitHubLoader):
  File "pydantic/main.py", line 198, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 663, in pydantic.fields.ModelField._type_analysis
  File "pydantic/fields.py", line 808, in pydantic.fields.ModelField._create_sub_type
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "/home/user/anaconda3/lib/python3.9/typing.py", line 852, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class
```

**Comment by ekiwi111** [comment]
Fixed with `pip install typing_extensions==4.5.0`

**Comment by jianghuyihei** [comment]
Thank you for your discovery. If you find any problems in the future, you can submit a pull request. Thank you for your contribution to the project
